awx_collection/plugins/modules/tower_ad_hoc_command.py

```diff
@@ -118,7 +118,7 @@ def main():
         limit=dict(),
         credential=dict(required=True),
         module_name=dict(required=True),
-        module_args=dict(default=""),
+        module_args=dict(),
```

Comment:
Can you confirm there is a test on this one? Down on like 148 module_args is used to build a post_data without a conditional `if module_args`

Reply:
I'm working on this one, have to make a test for it.

Reply:
Test added, does not look to affect the module.